Library/Formula/passenger.rb

```diff
@@ -23,6 +23,11 @@ def install
     rake "nginx"
     rake "webhelper"
 
+    # Fixes https://github.com/phusion/passenger/issues/1288
+    rm_rf "buildout/libev"
+    rm_rf "buildout/libeio"
+    rm_rf "buildout/cache"
```

Comment:
Hello FooBarWidget, I am having trouble locating this passenger file. Is this the gem file?